File: server/src/main/java/com/example/server/DomainForNewAccount/CarInfo.java
package com.example.server.DomainForNewAccount;

import java.io.File;

import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class CarInfo {
    private String licensenum;
    private String expirationdate;
    private String driverlicensefile;
    private String carnum;
}

File: client/employee-management/src/app/register/create-account-car-info/carInfo.ts
export class CarInfo {
  licensenum: string;
  expirationdate: string;
  driverlicensefile: string = '';
  carnum: string;

  constructor(
    licensenum: string = '',
    expirationdate: string = '',
    driverlicensefile: string = '',
    carnum: string = ''
  ) {
    this.licensenum = licensenum;
    this.driverlicensefile = driverlicensefile;
    this.expirationdate = expirationdate;
    this.carnum = carnum;
  }
}
